test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Metric/StartupWithMetricHandler.cs

```diff
@@ -21,7 +21,10 @@ public StartupWithMetricHandler(IConfiguration configuration)
         {
         }
 
+        [System.Obsolete]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
```

Comment:
why you need this obsolete attribute?

Reply:
While running E2E test, it was giving FAILED. I removed it :)